### DISCUSSION TITLE
Are DLCs imported?

### DISCUSSION THREAD

**Discussion by 51cent** [legendary-gl/legendary#262] [Q&A]
Hallo,

I just started using legendary, so the first thing I did was importing my already downloaded games. But only the base game is listed when running "list-installed". The DLCs are missing.

Example:
```
legendary.exe list-installed
...
 * Metro Exodus (App name: Snapdragon | Version: 0.1.0.38 | 70.08 GiB)
```

```
legendary.exe list-games
...
 * Metro Exodus (App name: Snapdragon | Version: 0.1.0.38)
  + Sam's Story (App name: SnapdragonDLC2 | Version: 0.1.0.6)
  + The Two Colonels (App name: SnapdragonDLC1 | Version: 0.1.0.18)
```

So I thought I would have to import the DLCs separetly using the same command only changing to the DLCs app name. But legendary says it is already installed:
```
legendary.exe import-game SnapdragonDLC2 "path-to\MetroExodus"
[cli] ERROR: Game is already installed!
```

I'm confused now if the "list-installed" command just does not list DLCs or if they actually have not been imported. The readme and help message are not clear on this.

Thanks in advance for the answer and keep up the great work!

**Comment by derrod** [comment]
`list-installed` currently does not list DLCs. It doesn't automatically import them either though. But if you ever ran an update or repair command then you probably installed them with that.
